ci/test_wheel.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -eou pipefail
+
+mkdir -p ./dist
```

Comment:
I'm wondering if this command is superfluous. @ajschmidt8 [we already have an `mkdir -p` in this tool](https://github.com/rapidsai/gha-tools/blob/main/tools/_rapids-download-from-s3#L14), that's the same directory that `rapids-download-wheels-from-s3` is ultimately using, right? If so, we could also remove this `mkdir -p` from all the wheel test scripts (future cleanup for all the merged PRs).

Reply:
I'm testing this in the rmm/cudf/cuspatial PRs too.

Reply:
So we just remove this line?